gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/iap/IAPSsoConfig.java

```diff
@@ -62,6 +64,19 @@
   @Autowired
   IAPSecurityConfigProperties configProperties;
 
+  @Bean
+  public IAPAuthenticationFilter iapAuthenticationFilter() {
+    return new IAPAuthenticationFilter(
+      configProperties, permissionService, front50Service);
+  }
+
+  @Bean
+  public FilterRegistrationBean registration(IAPAuthenticationFilter filter) {
```

Comment:
IIRC the bean name is taken from the method name, so you may want to use a more descriptive name here.

Reply:
Done.